src/core/modules/StarMgr.cpp

```diff
@@ -624,10 +631,11 @@ void StarMgr::loadData(QVariantMap starsConfig)
 	qInfo() << "Loading star data ...";
 
 	catalogsDescription = starsConfig.value("catalogs").toList();
+	bool hasFailed = false;
 	foreach (const QVariant& catV, catalogsDescription)
 	{
 		QVariantMap m = catV.toMap();
-		checkAndLoadCatalog(m);
+		hasFailed = not checkAndLoadCatalog(m, not hasFailed);
```

Comment:
May be better to avoid these lots of negations and just use `success` instead of `hasFailed`.

Reply:
fixed, thanks!